.circleci/config.yml

```diff
@@ -123,20 +125,38 @@ jobs:
             svn ci -m "Tagging ${BUILD_VERSION} from GitHub" --no-auth-cache --non-interactive --username "${SVN_USERNAME}" --password "${SVN_PASSWORD}"
 
 workflows:
-  test-deploy:
+  lint-all:
+    jobs:
+      - lint
+
+  test-php:
+    when:
+      or:
+        - git log --name-status --exit-code --format=oneline << pipeline.git.revision >>...<< pipeline.git.base_revision >> | grep -E '\.php$'
     jobs:
       - php-tests:
           matrix:
             parameters:
               php-version: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
+
+  test-js:
+    when:
+      or:
+        - or: [ git diff --name-only << pipeline.git.revision >> << pipeline.git.base_revision >> | grep '\.js$' ]
```

Comment:
This approach isn't working...

It looks like it's not using bash, the contents of `[]` are just treated as a string.

Reply:
Maybe a feature request to CircleCI to request that it allow bash inside `when` conditionals. Though I couldn't find a good way to make a request yet.